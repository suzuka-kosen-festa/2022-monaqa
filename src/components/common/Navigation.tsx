import tw from 'twin.macro'

const Navigation = tw.a`text-blue-600 hover:text-blue-800`

export { Navigation }
