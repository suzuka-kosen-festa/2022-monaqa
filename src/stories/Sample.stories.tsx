import type { ComponentStoryObj, ComponentMeta } from '@storybook/react'
import { Sample } from '../components/Sample'

type T = typeof Sample
type Story = ComponentStoryObj<T>

export default {
  component: Sample,
} as ComponentMeta<T>

export const Default: Story = {}
