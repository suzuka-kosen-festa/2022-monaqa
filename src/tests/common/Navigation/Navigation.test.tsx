import { composeStories } from '@storybook/testing-react'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import React from 'react'
import * as stories from '../../../stories/common/Navigation.stories'

const { Default } = composeStories(stories)

describe('(components/common) Navigation', () => {
  test('take snap shot', () => {
    const { container } = render(<Default />)
    expect(container).toMatchSnapshot()
  })
})
