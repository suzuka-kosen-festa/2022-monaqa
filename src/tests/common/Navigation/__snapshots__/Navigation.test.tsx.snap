// Vitest Snapshot v1

exports[`(components/common) Navigation > take snap shot 1`] = `
<div>
  <a
    class="css-1oumg6"
    href="#"
  >
    This is test
  </a>
</div>
`;
